kde-plasma/plasma-meta/plasma-meta-9999.ebuild

```diff
@@ -31,7 +31,6 @@ RDEPEND="
 	>=kde-plasma/ksshaskpass-${PV}:${SLOT}
 	>=kde-plasma/ksystemstats-${PV}:${SLOT}
 	>=kde-plasma/kwayland-${PV}:${SLOT}
-	>=kde-plasma/kwayland-integration-5.27.9:5
```

Comment:
Why do you drop it?

Reply:
The source is currently only QT 5

https://invent.kde.org/plasma/kwayland-integration

Reply:
https://invent.kde.org/plasma/kwayland-integration/-/merge_requests/61

Reply:
Rebase on top of HEAD, dropping this change, and make this commit only about DESCRIPTION and kde-apps/khelpcenter:6, then we can merge it.

Reply:
See also my comment from commit c79492bca3c28e690c114fac53b1feb5151ac58a.

May be made optional in the future but makes no sense right now.